src/test/java/org/nmfw/foodietree/domain/store/service/StoreSignUpServiceTest.java

```diff
@@ -0,0 +1,33 @@
+package org.nmfw.foodietree.domain.store.service;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.nmfw.foodietree.domain.store.entity.Store;
+import org.nmfw.foodietree.domain.store.repository.StoreMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+import static org.junit.jupiter.api.Assertions.*;
+@SpringBootTest
+class StoreSignUpServiceTest {
```

Comment:
db에 테스트 내용이 반영되지 않도록 처리가 필요할거 같아요!

```suggestion
@SpringBootTest
@Transactional
class StoreSignUpServiceTest {
```